### PR TITLE
fix: if phantom is locked, might not able to open its popup

### PR DESCRIPTION
**Pull request by UncleBill**
https://mask.atlassian.net/browse/MF-379

### CODE REVIEW

File: packages/plugins/Solana/src/UI/SNSAdaptor/index.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { Web3UI } from '../Web3UI'
 const sns: Plugin.SNSAdaptor.Definition = {
     ...base,
     async init(signal, context) {
-        setupStorage(context.createKVStorage('memory', StorageDefaultValue))
+        setupStorage(context.createKVStorage('persistent', StorageDefaultValue))
```
Comment (hanszhoou) [inline packages/plugins/Solana/src/UI/SNSAdaptor/index.tsx:11]: May I ask the reason for this change?

Comment (UncleBill) [reply packages/plugins/Solana/src/UI/SNSAdaptor/index.tsx:11]: To restore data in next browser session.

Comment (guanbinrui) [reply packages/plugins/Solana/src/UI/SNSAdaptor/index.tsx:11]: Better avoid doing this.

**Review by guanbinrui — APPROVED** [review]

File: packages/plugins/Solana/src/UI/SNSAdaptor/index.tsx (hunk 1 above)
Comment (Jack-Works) [reply packages/plugins/Solana/src/UI/SNSAdaptor/index.tsx:11]: If you're gonna store this, please make sure you only store the needed information and leave the rest in the memory.

**Pull request merged**